test/runtests.jl

```diff
@@ -10,6 +10,12 @@ xseries(plt, idx=length(plt.series_list)) = plt.series_list[idx].plotattributes[
 yseries(plt, idx=length(plt.series_list)) = plt.series_list[idx].plotattributes[:y]
 zseries(plt, idx=length(plt.series_list)) = plt.series_list[idx].plotattributes[:z]
 
+@testset "heatmap" begin
+   x = (1:3)m
+   @test heatmap(x*x', clims=(1m^2,2m^2)) isa Plots.Plot # todo
+   @test heatmap(1:3, (1:3)*m, x*x', clims=(1m^2,3m^2)) isa Plots.Plot
```

Comment:
Would it make sense to add a few tests like:
```julia
plt = heatmap(unitful_matrix, clims=unitful) 
@test only(plt.series_list).plotattributes[:series_color] isa ColorGradient

plt = plot(unitful_matrix)
@test length(plt.series_list) == nrows
@test plt.series_list[1].plotattributes[:series_color] isa RBGA
```
Or is this overengineered and digging to much into internals? I am torn.

Reply:
It's up to you of course, but all the other current tests are just of the `isa Plots` type and it would help me out to get this merged so I can move forward on a dependent project.  (I wasn't familiar with `only` or `RGBA` so I won't be much help with such tests.)

Reply:
Ok let's not test these details.

Reply:
Maybe we can add some more conversions here though, like
```julia
@test heatmap(x*x', clims=(1km^2,2km^2)) isa Plots.Plot
```